src/platform/forms-system/src/js/definitions/autosuggest.js

```diff
@@ -26,11 +26,12 @@ export const schema = {
  */
 export function uiSchema(label, getOptions, options = {}) {
   const validations = [];
-  if (!_.get('ui:options.freeInput', options)) {
+  if (!options['ui:options']?.freeInput) {
```

Comment:
Ooh, good call on the optional chaining. `get` was giving us that for free.

Reply:
⛓️⛓️⛓️